cmd/warmup.go

```diff
@@ -102,6 +102,7 @@ func sendCommand(cf *os.File, batch []string, count int, threads uint, backgroun
 }
 
 func warmup(ctx *cli.Context) error {
+	setup(ctx, 0)
```

Comment:
1

Reply:
If warmup with `-f` option, the positional argument is not necessary.

Reply:
lgtm